rotkehlchen/chain/zksync_lite/manager.py

```diff
@@ -860,4 +868,24 @@ def decode_undecoded_transactions(self, force_redecode: bool) -> int:
 
             self.decode_transaction(transaction, tracked_addresses)
 
+            if send_ws_notifications and tx_index % 10 == 0:
+                self.msg_aggregator.add_message(
+                    message_type=WSMessageType.EVM_UNDECODED_TRANSACTIONS,
+                    data={
+                        'evm_chain': self.ethereum_inquirer.chain_name,
```

Comment:
can you change here `evm_chain` to `chain` as we do in places where we support evm and evm like? Change it in all the places. Also in the `websockets.rst` file and update the docs there